core/templates/services/rte-helper.service.ts

```diff
@@ -98,8 +98,8 @@ angular.module('oppia').factory('RteHelperService', [
           // this service is migrated to Angular and NgbModal is used instead of
           // uibModal. Currently, these custom classes are used for correctly
           // stacking AngularJS modals on top of Angular modals.
-          windowClass: 'forced-modal-stack-over',
-          backdropClass: 'forced-modal-backdrop-stack-over',
+          windowClass: 'oppia-customization-arg-editor-modal',
+          backdropClass: 'oppia-customization-arg-editor-modal',
```

Comment:
These changes might have an unintended side effect. The `forced-modal-stack-over` class (which basically has a high z-index assigned to it in `core/templates/css/oppia.css`) was added as a fix for [this issue](https://github.com/oppia/oppia/issues/14621). Could you please take a look at the issue I linked and confirm it doesn't re-occur due to these changes?

For reference, here's [the PR that added that class to the modal instance](https://github.com/oppia/oppia/pull/14670).

Reply:
Also, the video link in the PR's description points back to the PR. Please fix it. Thanks!

Reply:
[Uploading Screencast from 02-03-23 10:51:17 PM IST.webm…]()
There is no impact of my change into https://github.com/oppia/oppia/issues/14621.

Reply:
@kashishbhandula I'm sorry but the video link still doesn't work (it just links back to the page). Please take a look and fix the link.

Reply:
Hi @nithinrdy , I am sorry it's my mistake.